.semaphore/semaphore.yml

```diff
@@ -12,18 +12,29 @@ blocks:
       - name: "Yarn & caching"
         commands:
           - checkout
-          - cache restore node-modules-$SEMAPHORE_GIT_BRANCH
+          - cache restore client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum yarn.lock),client-node-modules-$SEMAPHORE_GIT_BRANCH,client-node-modules-master
```

Comment:
I see this is taken from their example but not sure where the last two keys are coming from... let's go with it for now though